app_core/audio/sources.py

```
@@ -375,6 +383,265 @@ def _load_mp3_file(self) -> np.ndarray:
+                samples = np.array(audio.get_array_of_samples(), dtype=np.float32)
+
+                # Convert to mono if needed
+                if audio.channels == 2:
+                    samples = samples.reshape((-1, 2))
+                    samples = np.mean(samples, axis=1)
+
+                # Normalize
+                max_val = float(2 ** (audio.sample_width * 8 - 1))
+                samples = samples / max_val
+
+                return samples
```

Ruff (0.14.3)

Consider moving this statement to an `else` block

(TRY300)